docs/monitoring.md

```diff
@@ -1125,12 +1125,14 @@ This is the component with the largest amount of instrumented metrics
   - stages.failedStages.count
   - stages.skippedStages.count
   - stages.completedStages.count
-  - tasks.blackListedExecutors.count
+  - tasks.blackListedExecutors.count // deprecated use excludedExecutors instead
```

Comment:
Is it even worth mentioning in the documentation?

Reply:
I think it is as until we come up with different way of deprecating only way to tell users